client/src/pages/Login.jsx

```diff
@@ -5,58 +5,39 @@ import { LOGIN } from "../utils/mutations";
 import Auth from "../utils/auth";
 import { Button, Typography, Input, Card, FormHelperText } from "@mui/joy";
 import { useForm } from "react-hook-form";
-import { useSchemas } from "../hooks/useSchemas";
-import { yupResolver } from "@hookform/resolvers/yup";
-
-const styles = {
-  card: {
-    width: 400,
-    mx: "auto",
-    p: 2,
-    mt: 4,
-    my: 4,
-    borderRadius: "sm",
-    display: "flex",
-    flexDirection: "column",
-    gap: 2,
-    boxShadow: "lg",
-    backgroundColor: "lightblue",
-  },
-};
+import { styles } from "../styles/cardstyles";
 
 const Login = () => {
   const [login, { error }] = useMutation(LOGIN);
 
-  const { loginSchema } = useSchemas();
+  const { register, handleSubmit } = useForm();
 
-  const {
-    register,
-    handleSubmit,
-    formState: { errors },
-  } = useForm({ resolver: yupResolver(loginSchema) });
+  const handleLogin = (data) => {
+    Auth.login(data);
+    alert("Successfully logged in!");
+  };
 
```

Comment:
I'm trying to make the `try` `catch` blocks "do" less so I've made a separate function to handle the login. I'm not sure if I'm just kicking the can down the road though.